holmes/plugins/sources/prometheus/plugin.py

```diff
@@ -57,6 +59,10 @@ def __fetch_issues_from_api(self) -> List[PrometheusAlert]:
             "silenced": "false",
             "inhibited": "false",
         }
+        if self.label_filter:
+            params["filter"] = self.label_filter
```

Comment:
how does this work with a list? do you have an example? is it 'and' or 'or' filtering?

Reply:
It's AND. You filter on both, which is what we want here.